man/systemd.link.xml

```diff
@@ -133,7 +133,10 @@
         <term><varname>Type=</varname></term>
         <listitem>
           <para>A whitespace-separated list of shell-style globs matching the device type, as exposed by
-          <command>networkctl status</command>. If the list is prefixed with a "!", the test is inverted.
+          <command>networkctl list</command>. If the list is prefixed with a "!", the test is inverted.
+          Some valid values are "<literal>ether</literal>", "<literal>loopback</literal>", "<literal>wlan</literal>", "<literal>wwan</literal>".
+          Valid types are named either from the udev "<literal>DEVTYPE</literal>" attribute, or
```

Comment:
Nice! But, could you drop `"` around `<literal>` tag? These are redundant.

Reply:
Of course. Sorry I didn't see how these were added by the DocBook `literal` tag.